.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env"]
+}
```

Comment:
Missing a new line here. Are you using an editor which supports `.editorconfig` files? We use these in every repo to auto-configure basic editor behaviour.

Reply:
Done, I was missing the VSCode extension.